scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -0,0 +1,422 @@
+            print(f"  Loaded {len(stable_releases)} stable/RC releases from local files")
+
+        experimental_releases = fetch_experimental_releases(
+            client, variant, experimental_count
+        )
+        if not quiet:
+            print(f"  Fetched {len(experimental_releases)} experimental releases from API")
+
+        selected = stable_releases + experimental_releases
+        tasks = get_download_tasks(variant, selected, output_dir)
+        return selected, tasks
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)